arex-schedule-web-api/pom.xml

```diff
@@ -104,6 +104,11 @@
             <version>1.0.29</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.retry</groupId>
+            <artifactId>spring-retry</artifactId>
+            <version>1.2.4.RELEASE</version>
```

Comment:
No need to control the version internally, spring-boot pom will do the version control